Mathlib/Topology/Sheaves/LocalPredicate.lean

```diff
@@ -63,9 +63,33 @@ a `P : PrelocalPredicate T` consists of:
 -/
 structure PrelocalPredicate where
   /-- The underlying predicate of a prelocal predicate -/
-  pred : ∀ {U : Opens X}, (∀ x : U, T x) → Prop
+  pred : ∀ ⦃U : Opens X⦄, (∀ x : U, T x) → Prop
+  -- TODO: change `pred` to `Pred` according to naming convention
   /-- The underlying predicate should be invariant under restriction -/
-  res : ∀ {U V : Opens X} (i : U ⟶ V) (f : ∀ x : V, T x) (_ : pred f), pred fun x : U ↦ f (i x)
+  res : ∀ {U V : Opens X} (i : U ⟶ V) (f : ∀ x : V, T x), pred f → pred fun x : U ↦ f (i x)
+
+section Sheafify
+
+variable {T} (P : ∀ ⦃U : Opens X⦄, (∀ x : U, T x) → Prop)
+
+/-- The sheafification of a predicate. -/
+def Sheafify ⦃U : Opens X⦄ (f : ∀ x : U, T x) :=
```

Comment:
I'm not a huge fan of "Sheafify" for this.

Reply:
I'm just copying the name from [TopCat.PrelocalPredicate.sheafify](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Sheaves/LocalPredicate.html#TopCat.PrelocalPredicate.sheafify), and I think the name is intuitive. What do you not like about it?